roborazzi/src/main/java/com/github/takahirom/roborazzi/Roborazzi.kt

```diff
@@ -591,7 +591,7 @@ private fun saveOrCompare(
       true
```

Comment:
Does it make sense to rename the function `saveOrCompare` wrapping this logic given it could be saving and comparing at the same time?

Reply:
Thank you for your suggestion. I agree, the name 'processOutputImageAndReport' more accurately represents the function's behavior as it can both save and compare images. I will make this change.